app/src/store/modules/explorer/gallery/actions.js

```diff
@@ -0,0 +1,23 @@
+const placeholderDesc = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
+export default {
+  async loadItems ({ commit, getters }, { page = 1 } = {}) {
+    // Loads dummy items for now
+    if (page < 1 || (page > 1 && page > getters.totalPages)) {
+      throw new Error(`Invalid Page Number: ${page}. Must be number from 1 to ${getters.totalPages}`)
+    }
+    const startIndex = (page - 1) * getters.pageSize
+    const total = 333
```

Comment:
I'll suggest retrieving the total from the returned result. Also, can we connect and query the prod SPARQL endpoint to get the data we need pending the completion of the KG implementation? Just a heads-up, there is a reusable sparql.js.

Reply:
this was done using the "instances" view in whyis.  added a `whyis-views.js` with utility functions for constructing a whyis view URL and retrieving whyis views as json.